### ISSUE TITLE
How can I download the spirit/datomic library? 

### ISSUE THREAD

**Issue by abhi18av** [zcaudate/spirit#19]
Hi @zcaudate 

First off, I must tell you that I am a huge fan of your work - I'm learning a lot from you as a clojure beginner.

Now, coming back to the point - I was trying to use the `spirit.datomic` and `im.chit/adi` but neither of them is actually published on `clojars`.

Could you tell me how can I still use them or when are you planning to publish the complete `spirit` package ?

**Comment by zcaudate** [comment]
@abhi18av i thought it was published. Let me have a check and get back to you. The functionality in the library is being built on a needs basis and so it has been slow. I'm currently trekking in Nepal but will get back to working on it next month.

**Comment by abhi18av** [comment]
Hello neighbour :+1: 

Looking forward to this library as well

**Comment by zcaudate** [comment]
@abhi18av: yes, you're right. The latest version is not published - although https://clojars.org/im.chit/adi has not really changed. 

It's a little confusing right now but I'm hoping to solve it in the coming month. Can I ask what you are looking to do with it?

**Comment by abhi18av** [comment]
Well, I'd like to build my knowledge base as well as guide people and my organization into Clojure using the `hara` and `lucidity` packages.

If this one is published as a proper group of libs it could make the transition and learning much more fun :)

**Comment by zcaudate** [comment]
@abhi18av: Thanks for your interest in these libraries. 

I'll give a rough breakdown of the current state:

- In terms of `hara`, most of the elements are there, there is 99% test coverage and more than 95%  documentation so it is very stable and it's my lego collection of features for any basic system.

- In terms of `lucidity`, it does the job that I need it to do, which is to provide all the necessary infrastructure for publishing documentation and libraries for hara. More documentation is needed, especially around `lucid.unit` and `lucid.publish`. Also, `lucid.package` is basically a bare-bones leiningen/aether and the java port of encryption is for > gpg 2.0, which is a bit of a headache at the 
moment because I think I bit off more than I could chew. It does need a bit of TLC in order to get other people onboard but to be perfectly honest, I'm quite happy right now being the only person that is using it because it does what I need it to do.

- In terms of `spirit`, this is my very opinionated lego collection for building large systems (with ideas taken from many projects that I've worked on in the past). I'm a little bit behind the ball than what I thought previously. It has been designed/envisioned to be with `lucidity` for packaging/testing/documentation but I've left it alone for a couple of months now and it looks like it has not be done. I'm working on some personal projects at the moment and will be fleshing out the functionality. If you are interested, I'd love to have a chat about what you are looking for because it also helps me to design a better library.
